packages/react-lowcode/src/codegen/index.ts

```diff
@@ -31,13 +31,10 @@ export function generatePages(inputSourceCode: string, io: CodeRW & CodeDir, opt
                     getTypeText: () => prop.getDeclarations()[0].getText()
                 }))
             }
-            const context = {
-                useFormatter: true,
-                tableType: TableType.DataTable,
-                uiFramework: options.uiFramework ?? UiFramework.MaterialUI,
-                entity,
-            }
-            const generator = new ModuleGenerator(context)
+
+            let context = {uiFramework: UiFramework.MaterialUI, formatter: Formatter.None, index: {tableType: TableType.BasicTable, height: "400px"}};
```

Comment:
What about renaming formatter. For example formater -> localeFramework. I would like to avoid confusion (both source code formatter and localization message formatter sharing the same word formatter).